Resources/Prototypes/Roles/Jobs/Cargo/cargo_technician.yml

```diff
@@ -3,7 +3,7 @@
   name: job-name-cargotech
   description: job-description-cargotech
   playTimeTracker: JobCargoTechnician
-  antagAdvantage: 2 # DeltaV - Reduced TC: External Access
+  antagAdvantage: 2 # DeltaV - Reduced TC: Cargo Access
```

Comment:
yeah cargo techs cant buy things alone anymore (outside of lowpop ofc)

Reply:
I forgor to reduce the cargo advantage

The argument was because they could smuggle stuff out semi-easily, I removed it all together in a first draft